pkg/yang/yang.go

```diff
@@ -624,13 +624,13 @@ type Uses struct {
 	Parent     Node         `yang:"Parent,nomerge" json:"-"`
 	Extensions []*Statement `yang:"Ext" json:"-"`
 
-	Augment     *Augment  `yang:"augment" json:",omitempty"`
-	Description *Value    `yang:"description" json:",omitempty"`
-	IfFeature   []*Value  `yang:"if-feature" json:"-"`
-	Refine      []*Refine `yang:"refine" json:"-"`
-	Reference   *Value    `yang:"reference" json:"-"`
-	Status      *Value    `yang:"status" json:"-"`
-	When        *Value    `yang:"when" json:",omitempty"`
+	Augment     []*Augment `yang:"augment" json:",omitempty"`
```

Comment:
Should we create a different field here, since this means that this change is backwards-incompatible for existing API users. There was some discussion historically about making such changes for YANG 1.1. 

If we don't do this, then we need to mark this as goyang v2. I'd prefer to see whether we can group up other API changes that would be needed to fully support y1.1 into such a release.

Reply:
I'm unsure what exactly you want me to do here.

Reply:
I don't support changing `Augment` from `*Augment` to `[]*Augment` since this is backwards incompatible. This means that we're going to need to all downstream clients to change.

I suggest this is implemented using a new field `Augments` which has the slice. If it's not, then I don't think we should merge this without discussing other cardinality changes that are needed to support YANG 1.1 in a goyang v2.